example_batch.js

```diff
@@ -50,8 +50,8 @@ var myBatch = {
           "distance_unit": "in",
           "weight": "2",
           "mass_unit": "oz"
-        }]
-      }
+        }}
```

Comment:
It's a little hard to figure this out just based on the diff - what's the effect of changing the brackets here?

Reply:
It was missing square bracket above, has been corrected to create an array